main/security.bs

```diff
@@ -3,6 +3,33 @@ Security Considerations {#security}
 
 Issue: Write Security Considerations section.
 
+Solid data pods [disable all cross-origin protections](#cors-server) in browsers
+because resource access is governed explicitly by [Web Access Control](#wac).
+As such,
+data pods MUST NOT rely on cross-origin protection
+for shielding access to resources.
+While this ensures that unauthorized resource access will not occur,
+additional security measures MAY be needed
+to prevent metadata in error responses from leaking.
+For instance,
+a malicious app could probe multiple servers
+to check whether the response status code is `401` or `403`,
+or could try to access an error page
+from an intranet server within the user agent's private network
+to extract company names or other data.
+To mitigate this,
+when a request from an untrusted the `Origin` arrives,
+the data pod MAY set the status code of error responses to `404`
+and/or anonymize or censor their contents.
+
+A data pod MUST NOT assume that the user agent is a regular Web browser,
+even when requests contain familiar values
+in headers such as `User-Agent` or `Origin`.
+
+
 ## Privacy Considerations ## {#privacy}
 
 Issue: Write Privacy Considerations section.
+
+In order to prevent leakage of non-resource data,
+error responses SHOULD NOT contain identifiable information.
```

Comment:
When I'm reading something like this, I'd normally like to briefly know more, so would adding ", for example, .." make sense?

Perhaps preferably, it should refer to a reusable subsection on "Identifiable Information" under the "Privacy Consideration" section.

Also, why not a MUST NOT?

Reply:
> Perhaps preferably, it should refer to a reusable subsection on "Identifiable Information" under the "Privacy Considerations" section.

👍

> Also, why not a MUST NOT?

Because it might be hard to determine what identifiable information is (an extra space in the source code that only occurs with v37.466 which they run at place X), and to entirely get rid of it. And maybe people don't mind.

Reply:
True. Perhaps once we determine if and what is normatively and/or non-normatively mentioned as identifiable information, it can clarify this point. The normative items need a test, and so MUST NOT works here. Otherwise, SHOULD NOT is fine as a catchall. I think?